src/mots/utils.py

```diff
@@ -39,3 +39,19 @@ def parse_real_name(real_name):
         return match.groupdict()
     else:
         return {"name": None, "info": None}
+
+
+def mkdir_if_not_exists(path):
+    """Check if a directory exists, if not, create it."""
+    if not path.exists():
+        path.mkdir()
+    elif path.exists() and not path.is_dir():
+        logger.warning(f"{path} exists but is not a directory.")
```

Comment:
Can the program continue without failing later if we hit this code path? If we are expecting `path` to be a directory or not exist maybe this should throw.

Reply:
Good point -- the code will likely fail elsewhere, however that would be out of scope for this method. I do think it makes sense to add a check perhaps in `cli` to deal with a situation where the resource directory can not be created (or is a file) and perhaps fail gracefully. Same with the overrides file. Filed [a bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1763168) for this.